astropy/utils/console.py

```diff
@@ -1150,35 +1130,3 @@ def __call__(self):
         import msvcrt
 
         return msvcrt.getch()
-
-
-class _GetchMacCarbon:
```

Comment:
There's a related obsolete comment in `_GetchUnix`:
https://github.com/astropy/astropy/blob/febf980558a51df3992e083716b869231c9a3007/astropy/utils/console.py#L1125-L1126

Reply:
good catch